portal-api/.eslintrc.js

```diff
@@ -38,16 +33,22 @@ module.exports = {
       files: ['*.{j,t}s'],
       rules: {
         'class-methods-use-this': 'off',
-        'max-len': ['error', 160, 2, {
-          ignoreUrls: true,
-          ignoreComments: false,
-          ignoreRegExpLiterals: true,
-          ignoreStrings: true,
-          ignoreTemplateLiterals: true,
-        }],
+        'max-len': [
+          'error',
+          160,
+          2,
+          {
+            ignoreUrls: true,
+            ignoreComments: false,
+            ignoreRegExpLiterals: true,
+            ignoreStrings: true,
+            ignoreTemplateLiterals: true,
+          },
+        ],
         'import/no-unresolved': 'error',
         'no-console': ['error', { allow: ['info', 'error'] }],
         'no-underscore-dangle': ['error', { allow: ['_id', '_csrf'] }],
+        'no-param-reassign': ['error', { props: true, ignorePropertyModificationsForRegex: ['^draft', 'req', 'res'] }],
```

Comment:
`draft` is for immer to work (with `^` for wildcard naming)
`req` and `res` to match `main-stbr`